.buildkite/scripts/run-accessibility-check.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+
+yarn install --frozen-lockfile
+
+yarn storybook:build
+yarn ts-node storybook/bin/check-storybook-accessibility.ts
```

Comment:
TODO
This still needs a base storybook url passed in as an argument.
Need to experiment with passing in a file path, so that `public/storybook` works.

Reply:
Done in https://github.com/cultureamp/kaizen-design-system/pull/501/commits/d8482f13eb9a32ab60465f5c5c0a27199bd0088e